addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -181,7 +183,7 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
         const { data } = this.state;
         const $dropdown = this.$('.o_knowledge_icon > .o_article_emoji_dropdown');
         $dropdown.attr('data-article-id', this.state.res_id);
-        $dropdown.find('.o_article_emoji').text(data.icon);
+        $dropdown.find('.o_article_emoji').text(data.icon || '');
```

Comment:
Shoudln't icon be initialized with '' if there is an issue with false ?

Reply:
see other comment on previous commit

Reply:
My question was more: shouldn't we initialize icon in data.icon with '' when it is false ? That way we initialize variables with clean values, instead of having to have fallbacks everywhere. Not sure if doable (don't know the whole internals of code), just proposing.